src/SandboxCoordinator.ts

```diff
@@ -60,12 +60,13 @@ export default class SandboxCoordinator {
   }
 
   private createSandboxes(): Promise<Sandbox[]> {
-    const cpuCount = os.cpus().length;
+    const numConcurrentRunners = this.options.maxConcurrentTestRunners || os.cpus().length;
```

Comment:
The default value for `maxConcurrentTestRunners` is [`Infinity`](https://github.com/stryker-mutator/stryker-api/blob/master/src/config/Config.ts#L20). I think this is also why the travis build is failing. 

I think a better implementation could be:

```typescript
 let numConcurrentRunners = os.cpus().length;
let source = 'CPU count';
if(numConcurrentRunners > this.options.maxConcurrentTestRunners && this.option.maxConcurrentTestRunners > 0){
   numConcurrentRunners = this.options.maxConcurrentTestRunners;
   source = 'maxConcurrentTestRunners';
}
```

or something like that